app/src/androidTest/java/com/example/campusguide/SwitchCampusSystemTest.kt

```diff
@@ -50,7 +50,7 @@ class SwitchCampusAcceptanceTest {
     fun clickSwitchCampusButtonOnce() {
 
         // Wait until the map is loaded
-        device.wait(Until.hasObject(By.desc("Google Maps Ready")), TIMEOUT)
+        device.wait(Until.hasObject(By.desc("maps_activity_map ready")), TIMEOUT)
```

Comment:
Given how many places had to be refactored to accomodate changing this destriptor, i think we need to move it into a constant or something. I know it changes based on which map it is, but we still need to make it easier to adjust without breaking stuff.

Reply:
Done